src/nss-systemd/userdb-glue.c

```diff
@@ -129,6 +129,85 @@ enum nss_status userdb_getpwuid(
         return NSS_STATUS_SUCCESS;
 }
 
+int nss_pack_user_record_shadow(
+                UserRecord *hr,
+                struct spwd *spwd,
+                char *buffer,
+                size_t buflen) {
+
+        const char *hashed;
+        size_t required;
+
+        assert(hr);
+        assert(spwd);
+
+        assert_se(hr->user_name);
```

Comment:
Why `assert_se()` instead of `assert()`?

Reply:
no reason besides copy/paste sloppiness ;-)